<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>762</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Чат-бот</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/new/images/33.png</normaloff>:/new/images/33.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layout_chat_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>381</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <layout class="QVBoxLayout" name="layout_chat">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layout_details" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>491</width>
      <height>104</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="btn1_1">
         <property name="text">
          <string>Кому подойдёт</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn1_2">
         <property name="text">
          <string>Язык</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="btn1_3">
         <property name="text">
          <string>Стоимость</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn1_4">
         <property name="text">
          <string>С чего начать</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btn1_back">
       <property name="text">
        <string>Назад</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layout_letter" native="true">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>500</y>
      <width>491</width>
      <height>65</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="btn1_4_1">
       <property name="text">
        <string>Мотивационное письмо</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn1_4_back">
       <property name="text">
        <string>Назад</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layout_patners" native="true">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>180</y>
      <width>491</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QPushButton" name="btn2_1">
         <property name="text">
          <string>Универ1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn2_2">
         <property name="text">
          <string>Универ2</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QPushButton" name="btn2_3">
         <property name="text">
          <string>Универ3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn2_4">
         <property name="text">
          <string>Универ4</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="btn2_5">
         <property name="text">
          <string>Универ5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn2_6">
         <property name="text">
          <string>Универ6</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btn2_back">
       <property name="text">
        <string>Назад</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layout_main" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>491</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_16">
       <item>
        <widget class="QPushButton" name="btn1">
         <property name="text">
          <string>Что такое АМ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn2">
         <property name="text">
          <string>Университеты-партнёры</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_17">
       <item>
        <widget class="QPushButton" name="btn4">
         <property name="text">
          <string>Контакты</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn3">
         <property name="text">
          <string>Желаете присоедениться?</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layout_enter" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>600</y>
      <width>489</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Enter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1039</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Настройка</string>
    </property>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>О программе</string>
    </property>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Как пользоваться...</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Оценить приложение</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Детальнее</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Настройка</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Нашли баг?</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
